lib/app/index.coffee

```diff
@@ -42,6 +42,17 @@ module.exports = ->
     trackSignup('email'),
     ssoAndRedirectBack
 
+  # Apple OAuth
+  app.get opts.applePath, setCampaign, beforeSocialAuth('apple')
+  app.get opts.appleCallbackPath,
```

Comment:
Do we still need both `get` and `post` here?

Reply:
Still looking into whether it is necessary, but I am thinking no.

Reply:
Seems like a bug in the passport-apple library, both endpoints were being called when only post should have been. Using a different package now that seems to work as expected 👍